doc/site/content/docs/guides/resource-excess/_index.md

```diff
@@ -0,0 +1,57 @@
++++
+title = "Resource excess"
```

Comment:
Sounds a bit weird to have an explanation specifically for excess and not the rest of the resource system, but I guess there's specific demand for it due to ongoing BAR efforts so it's probably okay from that angle.